### PR TITLE
Expand the Studio example gallery

### PR DESCRIPTION
**Pull request by yjsoon**
## Summary

- add four original classroom-ready examples to balance science, humanities and languages across upper-primary and secondary levels
- organise the 15-example iPad gallery by subject and learner level, while retaining activity type as a secondary filter
- enforce five-subject/two-level corpus coverage and keep unexpected future metadata filterable
- localise Malay player chrome across supported interactions and ready-made tools, with correct language semantics for fallback accessibility labels

## Verification

- `npm run verify:studio` — passed (18 spec tests, 25 player tests, 57 API tests, typechecks, build, 18-example corpus and iPad resource parity)
- iPad unit suite — 66 passed
- focused Explore launch/filter/reset UI tests — 2 passed
- actual Malay example preview UI test — passed on iPad (A16) simulator
- `git diff --check` — passed

## Adversarial review

Three independent xhigh Codex reviewers completed multiple adversarial passes before this PR was opened, covering correctness/regressions, tests/accessibility/edge cases, and UX/scope/localisation risk. Review fixes included preserving unknown filter metadata, strengthening corpus guarantees, exposing activity selection to VoiceOver, removing brittle catalogue-size assertions, localising Malay player chrome, and separating authored/fallback languages in accessible names. Final review of commit `407ee51` reported no actionable findings.

## Summary by CodeRabbit

* **New Features**
  * Added filtering of example activities by subject, learner level, and activity type.
  * Added Malay-language support across widget instructions, controls, feedback, reports, and ready-made activities.
  * Added four new interactive learning activities covering communication, geography, science, and environmental systems.
  * Improved accessibility and language labeling throughout the widget player.

* **Updates**
  * Refined perimeter and catchment simulation questions to better reflect their underlying concepts.
  * Expanded the example activity collection across subjects and learner levels.

### CODE REVIEW

File: examples/studio/fixed-perimeter-rectangle-explorer.widget.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
             "variableId": "width-value",
             "label": "Width",
             "minimum": 1,
-            "maximum": 5,
+            "maximum": 19,
```
[the remark this answers is not in the document]
Comment (yjsoon) [reply examples/studio/fixed-perimeter-rectangle-explorer.widget.json]: Fixed in f5bc206. The plot domain is back to 1–5 cm, which remains valid even at the minimum 12 cm perimeter, and the regenerated iPad catalogue plus full Studio verifier pass.

File: apps/widget-player/src/localisation.ts [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  },
+  sending: 'Sedang menghantar…',
+  sendReport: 'Hantar laporan',
+  reportFailed: 'Laporan tidak dapat dihantar. Semak sambungan anda dan cuba lagi.',
+  noPageTitle: 'Widget ini tidak mempunyai halaman untuk dipaparkan',
+  noPageMessage: 'Minta guru menerbitkan semula aktiviti, kemudian buka pautan sekali lagi.',
+};
+
+export function playerCopy(locale?: string): PlayerCopy {
+  return locale?.toLowerCase().startsWith('ms') ? malay : english;
```
[the remark this answers is not in the document]
Comment (yjsoon) [reply apps/widget-player/src/localisation.ts]: Fixed in 6c84e1d. Locale selection now compares the exact primary language subtag, and the same helper is reused for player copy, labels and component fallback semantics. A schema-valid msa regression case now confirms English fallback while preserving the authored document language; full Studio verification and resource parity pass.

File: apps/widget-player/src/components.tsx [hunk 3]
```diff
@@ -889,41 +922,49 @@ function ItemContentRenderer({
...
+          <span className="visually-hidden" id={descriptionId}>{altText}.</span>
+          <span className="visually-hidden" id={fallbackId} lang={copy.locale}>{copy.imageUnavailableShort}.</span>
+        </>
+      ) : null}
       <span aria-hidden="true">▧</span>
-      {!decorative ? <small>Image unavailable</small> : null}
+      {!decorative ? <small aria-hidden="true" lang={copy.locale}>{copy.imageUnavailableShort}</small> : null}
     </div>
   );
 }
```
[the remark this answers is not in the document]
Comment (yjsoon) [reply apps/widget-player/src/components.tsx:962]: Fixed in c14b3a5. MissingImage now receives spec.metadata.locale and applies it explicitly to the authored alt-text node, while fallback copy retains its resolved player-copy language. Full Studio verification and regenerated iPad resource parity pass.
